libs/zune-png/src/lib.rs

```diff
@@ -162,6 +162,7 @@ extern crate alloc;
 extern crate core;
 
 #[cfg(feature = "std")]
+pub use apng::ActlChunk;
```

Comment:
adding a line before one protected by conditional compilation looks suspicious

Reply:
Oh my god, I am so sorry. the rust language server did that

Reply:
https://github.com/makepad/makepad/pull/654/commits/e4d501d114b9d5d8dce00a760adbef037106eb4a fixes this